packages/language-server/src/plugins/html-data.ts

```diff
@@ -180,6 +180,26 @@ export const astroAttributes = newHTMLDataProvider('astro-attributes', {
 				},
 			],
 		},
+		{
+			name: 'transition:animate',
+			description: 'Specifies an animation to use with this element on page transition',
```

Comment:
```suggestion
			description: 'Specifies an animation to use with this element on page transition.',
```

Reply:
None of the other descriptions have periods. Sure you want one here?